src/components/FeaturedSearch/FeaturedSearch.jsx

```diff
@@ -0,0 +1,143 @@
+        { id: 4, name: 'Product 4', IMAGE: imageDayLeo },
+        { id: 5, name: 'Product 5', IMAGE: imageBachBo },
+    ];
+
+    return (
+        <Stack direction="column" alignItems="center" spacing={2} sx={{ width: "100%", mt: "2.75rem" }}>
+            <Typography sx={{ fontSize: "2.1875rem", color: "#214400", fontWeight: "500" }}>TÌM KIẾM NỔI BẬT</Typography>
+            <Grid container rowSpacing="3rem" columnSpacing="1.69rem" width="72.69rem">
+                {products.map((product, idx) => (
+                    <Grid item xs={product.id <= 2 ? 6 : 4} key={product.id}>
+                        <Box pt="1.31rem" sx={{ position: "relative", height: product.id <= 2 ? '22.5rem' : '14.875rem', backgroundColor: "#F4FFEB", borderRadius: "0.625rem", cursor: "pointer", display: "flex", flexDirection: "column", alignItems: "center" }}>
+                            <motion.p
+                                style={
+                                    {
+                                        color: "#214400",
+                                        fontSize: product.id <= 2 ? "1.5625rem" : '1.25rem',
+                                        fontWeight: "500",
+                                        fontFamily: " Roboto Serif",
+                                        transition: "all 0.1s",
+                                        opacity: hoverIndex == idx ? "0" : "1"
+                                    }
+                                }
+
+                            >
+                                {product.name}
+                            </motion.p>
+                            <motion.div
+                                style={
+                                    {
+                                        width: product.id <= 2 ? "32.875rem" : "21.4375rem",
+                                        height: product.id <= 2 ? "19.5625rem" : "12.9375rem",
+                                        boxShadow: "0px 6px 6px -3px rgba(0,0,0,0.2), 0px 10px 14px 1px rgba(0,0,0,0.14), 0px 4px 18px 3px rgba(0,0,0,0.12)",
+                                        borderRadius: "10px",
+                                        position: "absolute",
+                                        bottom: product.id <= 2 ? "-1.38rem" : "-1.12rem",
+                                        overflow: "hidden"
+                                    }
+                                }
+                                whileHover={{ y: "-14%" }}
+                                onMouseOver={() => handleHover(idx)}
+                                onMouseLeave={handleLeave}
+                            >
+
+                                <img src={product.IMAGE} alt="" style={{ height: "100%", width: "100%", objectFit: "cover", borderRadius: "10px" }} />
+
+                                {/* set hover and show text background black */}
+                                {hoverIndex === idx &&
+                                    <motion.div
```

Comment:
You can use the `framer-motion` in the MUI tags with the `component` prop instead

You can read the details here: [Using framer-motion into MUI tags](https://stackoverflow.com/a/65681968)